flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/AggregateITCase.scala

```diff
@@ -1334,4 +1334,32 @@ class AggregateITCase(
     val expected = Seq("3,29.39,tom123@gmail.com")
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
+
+  @Test
+  def testSplitAggsHandler(): Unit = {
+
+    val t = env.fromCollection(TestData.smallTupleData3)
+      .toTable(tEnv, 'a, 'b, 'c)
+    tEnv.createTemporaryView("MyTable", t)
+
+    tEnv.getConfig.setMaxGeneratedCodeLength(1)
+
+    val columnNumber = 100
```

Comment:
I would like to update this a bit. I tried to revert the changes and this test is still passed. That means this test can't reproduce the 64K problem. I changed to number up to `500` and reproduced this problem. Besides, I would like to remove `tEnv.getConfig.setMaxGeneratedCodeLength(1)` to make sure it works well with out-of-box configuration.

Reply:
OK, will change it.